src/middleware/validateApiKey.js

```diff
@@ -64,28 +24,16 @@ export default async (req, res, next) => {
     if (!apiKey && isDevelopment) {
       apiKey = FALLBACK_API_KEY;
     }
-    if (!apiKey) {
-      throw new Error('X-API-Key Header doesn\'t exist');
-    }
```

Comment:
revert this change - we don't want to alter existing logic

I'm thinking this file shouldn't even be changed since there are changes to code used actively in production. feel free to reach out to me to get more clarification in you need some 🙇🏾‍♀️